pipeline/src/lib.rs

```diff
@@ -5,7 +5,7 @@
 
 //! # Pipeline Building Blocks
```

Comment:
I think we need all `#[deny(clippy:all), deny(clippy:pendantic]` since we are no longer inheriting it.